examples/deepResearch/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@gensx-examples/deep-research",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "engines": {
+    "node": ">=18.0.0"
+  },
+  "scripts": {
+    "dev": "nodemon",
+    "run": "NODE_OPTIONS='--enable-source-maps' tsx ./index.tsx",
+    "build": "tsc",
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix",
+    "test": "NODE_OPTIONS='--enable-source-maps' tsx ./index.tsx"
+  },
+  "dependencies": {
+    "@gensx/openai": "workspace:*",
+    "@mendable/firecrawl-js": "^1.12.0",
+    "@types/node-fetch": "^2.6.12",
+    "gensx": "workspace:*",
+    "node-fetch": "^3.3.2",
+    "openai": "^4.77.0",
+    "xml2js": "^0.6.2"
+  },
+  "devDependencies": {
+    "@types/node": "^20.17.11",
+    "@types/xml2js": "^0.4.14",
+    "nodemon": "^3.1.9",
+    "prettier": "^3.0.0",
```

Comment:
I notice it is in the other examples as well, but I don't think this is necessary, as we have it in the root package.json.

Reply:
ahh makes sense. I've just been copying example folders and updating them so that explains the proliferation. I'll remove it in this one and go clean up the others at some point.